frontend/pages/item/[id]/index/edit.vue

```diff
@@ -67,14 +67,20 @@
 
     let purchasePrice = 0;
     let soldPrice = 0;
+    let purchaseTime = null;
     if (item.value.purchasePrice) {
       purchasePrice = item.value.purchasePrice;
     }
     if (item.value.soldPrice) {
       soldPrice = item.value.soldPrice;
     }
+    if (item.value.purchaseTime) {
+      purchaseTime = new Date(item.value.purchaseTime.getTime() - item.value.purchaseTime.getTimezoneOffset() * 60000);
```

Comment:
Why is a minute being tacked on at the end of this math?

Reply:
gettime returns number of milliseconds whereas gettimezoneoffset returns number of minutes, so multiplying the timezone offset by 60000 converts it to milliseconds
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset

Reply:
OK this makes sense, but also... What in the hell where the people who designed and implemented JavaScript drinking when they put this crap together? Both of those functions come from the same type for crying out loud!